packages/core/lib/commands/debug/mergeConfigNetwork.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (config, options) {
+  const inlineConfigNetwork = "inline_config";
+  config.networks[inlineConfigNetwork] = {
+    url: options.url,
+    network_id: "*"
+  };
+  config.network = inlineConfigNetwork;
+  config.merge(options);
```

Comment:
So, this works, because `config.merge()` does in fact modify `config` (something I'd forgotten until just now... yikes), but for consistency with our other code I think should actually return the modified config.